Command/QueryCommand.php

```diff
@@ -66,10 +64,10 @@ public function execute(InputInterface $input, OutputInterface $output)
                 $this->truncate($document->getDescription(), 50),
                 $document->getUrl(),
                 $document->getImageUrl(),
-                $document->getClass()
+                $document->getClass(),
             ));
         }
-        $table->render($output);
+        $table->render();
```

Comment:
when there is no results, I prefer a simple msg like There are no results available for the given query" instead of rendering an empty table.